MalApi/Anime/UpdateAnime.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace MalApi
+{
+    /// <summary>
+    /// The update object sent to MAL when updating an anime entry.
+    /// Only specified values will be changed. The rest will remain unchanged.
+    /// More details: https://myanimelist.net/modules.php?go=api#animevalues
+    /// </summary>
+    [XmlRoot("entry")]
+    public class UpdateAnime : UpdateObjectBase, IXmlSerializable
```

Comment:
Class names should be nouns, so this and UpdateManga should be AnimeUpdate/MangaUpdate.

Reply:
I thought I had commented on this yesterday, I guess I missed it. Update functionality should use System.Xml.Linq for XML generation instead of System.Xml.Serialization:

- Consistency. The MalAppInfoXml class already uses System.Xml.Linq.
- It avoids adding a dependency on System.Xml.XmlSerializer to the nuget package.
- It avoids putting implementation details in the API surface of MalApi such the XML-related attributes and the UpdateObjectBase class.

The method for creating XML should be internal, not public, and would look something like this using System.Xml.Linq.

```
            XDocument document = new XDocument(
                new XDeclaration("1.0", "UTF-8", null),
                new XElement("entry",
                    new XElement("Episode", Episode),
                    new XElement("Status", (int?)Status),
                    new XElement("Score", Score),
                   ...
                )
            );

            using (Utf8StringWriter writer = new Utf8StringWriter())
            {
                document.Save(writer);
                return writer.ToString();
            }
```